### PR TITLE
Deleted deprecated server classes and tests

### PR DESCRIPTION
**Pull request by JAmmermann-DLR**
Deleted the following classes:
- `Model`
- `VirSatGitAccess` and `VirSatGitAccessTest`
- `AccessTestResource` and `AccessTestResourceTest`
- `WorkspaceAccessResource` and `WorkspaceAccessResourceTest`
- `WorkspaceUserContext` and `WorkspaceUserContextTest`

Also removed now unused git repository from `AGitAndJettyServerTest` and renamed to `AJettyServerTest`

Closes https://github.com/virtualsatellite/VirtualSatellite4-Core/issues/597

### CODE REVIEW

File: de.dlr.sc.virsat.server.test/src/de/dlr/sc/virsat/server/dataaccess/TransactionalJsonProviderTest.java [hunk 1]
```diff
@@ -63,7 +63,6 @@
 
 	@Before
 	public void setUp() throws Exception {
-		super.setUp();
```
Comment (SaMuellerDLR) [inline de.dlr.sc.virsat.server.test/src/de/dlr/sc/virsat/server/dataaccess/TransactionalJsonProviderTest.java:66]: This is a bit strange. If we dont need the setup from the super class, then we are we extending AServerRepositoryTest?

**Review by SaMuellerDLR — CHANGES_REQUESTED** [review]
Very minor detail.

File: de.dlr.sc.virsat.server.test/src/de/dlr/sc/virsat/server/dataaccess/TransactionalJsonProviderTest.java (hunk 1 above)
Comment (JAmmermann-DLR) [reply de.dlr.sc.virsat.server.test/src/de/dlr/sc/virsat/server/dataaccess/TransactionalJsonProviderTest.java:66]: You are right that is strange. I looked into it and it called the `setUp()` from the `AGitAndJettyServerTest`, that `AServerRepositoryTest `extends, that I deleted in this PR. The problem causing this misunderstanding was wrong naming of the   junit setup/teardown methods in `AServerRepositoryTest`, so I will rename them to make it clear.

**Review by SaMuellerDLR — APPROVED** [review]
Approved!

**Review by andriikovalov-dlr — APPROVED** [review]

**Pull request merged**